DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/SelfAssessmentOverview.cshtml

```diff
@@ -62,13 +62,13 @@
     questionsTotal += questions.Count();
     <div class="nhsuk-panel competency-group-panel nhsuk-u-padding-0">
       @{
-          var expanderOpen = Model.CompetencyGroups.Count() == 1
+          var detailsOpen = Model.CompetencyGroups.Count() == 1
           ? "open"
           : ViewContext.RouteData.Values.ContainsKey("competencyGroupId")
             ? ViewContext.RouteData.Values["competencyGroupId"].Equals(competencyGroup.First().CompetencyGroupID.ToString()) ? "open" : ""
             : "";
       }
-      <details class="nhsuk-details nhsuk-expander" @(expanderOpen)>
+      <details class="nhsuk-details nhsuk-details" @(detailsOpen)>

```

Comment:
The competency group should continue to use the nhsuk-expander. The difference between them is explained in the NHS design system - the expander is used when most people will need to view its content, the details implementation is used when viewing detail is optional and not all users will want to see it. We should revert this change on line 71.